obspy/io/reftek/tests/test_core.py

```diff
@@ -0,0 +1,390 @@
+        #   11 0 848 2015-10-09T22:51:02.585000Z
+        #   12 1 790 2015-10-09T22:51:02.585000Z
+        #   13 2 844 2015-10-09T22:51:03.805000Z
+        #   14 0 892 2015-10-09T22:51:06.215000Z
+        #   15 1 768 2015-10-09T22:51:05.925000Z
+        #   16 2 884 2015-10-09T22:51:08.415000Z
+        #   17 1 778 2015-10-09T22:51:10.765000Z
+        #   18 0 892 2015-10-09T22:51:11.675000Z
+        #   19 2 892 2015-10-09T22:51:12.835000Z
+        #   20 1 736 2015-10-09T22:51:14.655000Z
+        #   21 0 892 2015-10-09T22:51:16.135000Z
+        #   22 2 860 2015-10-09T22:51:17.295000Z
+        #   23 1 738 2015-10-09T22:51:18.335000Z
+        #   24 0 892 2015-10-09T22:51:20.595000Z
+        #   25 1 673 2015-10-09T22:51:22.025000Z
+        #   26 2 759 2015-10-09T22:51:21.595000Z
+        #   27 0 067 2015-10-09T22:51:25.055000Z
+
+    def _assert_reftek130_test_stream(self, st_reftek):
+        """
+        Test reftek 130 data read into a stream object against miniseed files
+        converted using "rt_mseed" utility from Trimble/Reftek.
+
+        Note that rt_mseed fills in network as "XX", location as "01" and
+        channels as "001", "002", "003".
+        """
+        st_mseed = obspy.Stream()
+        for file_ in self.mseed_files:
+            st_mseed += obspy.read(file_, "MSEED")
+        # reftek reader correctly fills in band+instrument code but rt_mseed
+        # does not apparently, so set it now for the comparison
+        for tr in st_mseed:
+            tr.stats.channel = "EH" + tr.stats.channel[-1]
+            tr.stats.pop("_format")
+            tr.stats.pop("mseed")
+        # check reftek130 low-level headers separately:
+        for tr in st_reftek:
+            self.assertTrue("reftek130" in tr.stats)
+            # XXX TODO check reftek specific headers
+            tr.stats.pop("reftek130")
+            tr.stats.pop("_format", None)
+        # sort streams
+        st_reftek = st_reftek.sort()
+        st_mseed = st_mseed.sort()
+        # check amount of traces
+        self.assertEqual(len(st_reftek), len(st_mseed))
+        # check equality of headers
+        for tr_got, tr_expected in zip(st_reftek, st_mseed):
+            self.assertEqual(tr_got.stats, tr_expected.stats)
+        # check equality of data
+        for tr_got, tr_expected in zip(st_reftek, st_mseed):
+            np.testing.assert_array_equal(tr_got.data, tr_expected.data)
+
+    def test_read_reftek130(self):
+        """
+        Test original reftek 130 data file against miniseed files converted
+        using "rt_mseed" utility from Trimble/Reftek.
+
+        rt_mseed fills in network as "XX", location as "01" and channels as
+        "001", "002", "003".
+        """
+        st_reftek = _read_reftek130(
+            self.reftek_file, network="XX", location="01",
+            component_codes=["1", "2", "3"])
+        # centralized the equality check, so that it can be reused in other
```

Comment:
Not sure why this comment is here and not in other places, but it kind of seems obvious...